src/ansys/grantami/jobqueue/_models.py

```diff
@@ -138,61 +138,101 @@ def to_dict(self) -> Dict[str, Union[int, None]]:
         }
 
 
-class _JobFile:
+class JobFile:
     """
-    Represents a file associated with a job request.
+    Represents a file associated with a JobRequest.
 
     Parameters
     ----------
-    file_type : _FileType
-        Type of file being represented.
+    path : str or pathlib.Path
+        Path to the local file to use in a JobRequest. Can be absolute or relative.
+    virtual_path : str or pathlib.Path
+        Virtual path to use to refer to the file in the JobRequest. Must be relative.
```

Comment:
Should we spell out a bit more clearly that this is the path you should use in your Excel file?

Reply:
Indeed, there needs to be some documentation around how this affects attachments, I'll look into adding this as an example in docstrings. I had also talked with Esther about adding some content on that topic in the MI docs, so that we can link to it, this being similar to how jobs need to be setup with the import app

Reply:
I've added an example with attachment file in the docstring of JobFile. If you think that's insufficient, we can create a new notebook that illustrates working with attachments, files in "inaccessible" locations (network, or not in current directory) and how JobFile helps with this.